docs/src/config/sidebars.ts

```diff
@@ -413,6 +413,15 @@ const sidebars: SidebarsConfig = {
             }
           ]
         },
+        {
+          multitenant: [
+            {
+              type: 'doc',
+              label: 'multitenant add',
+              id: 'cmd/entra/multitenant/multitenant-add'
+            }
+          ]
+        },
```

Comment:
The position isn't alphabetical. This section should be placed after `m365group`

Reply:
Yes, I will wait until #6029 is merged.